tardis/io/config_validator.py

```diff
@@ -353,25 +360,33 @@ def check_type(self, value):
             return False
 
     def to_type(self, value):
-        quantity_split = value.strip().split()
-        quantity_value = quantity_split[0]
-        quantity_unit = ' '.join(quantity_split[1:])
-        if quantity_unit.strip() == 'log_lsun':
-            quantity_value = 10 ** (float(quantity_value) +
-                                    np.log10(constants.L_sun.cgs.value))
-            quantity_unit = 'erg/s'
+        if hasattr(value, 'unit') and hasattr(value, 'value'):
+            return value
+        try:
+            quantity_split = value.strip().split()
+            quantity_value = quantity_split[0]
+            quantity_unit = ' '.join(quantity_split[1:])
+            if quantity_unit.strip() == 'log_lsun':
+                quantity_value = 10 ** (float(quantity_value) +
+                                        np.log10(constants.L_sun.cgs.value))
+                quantity_unit = 'erg/s'
+
+            return float(quantity_value) * units.Unit(quantity_unit)
+        except:
```

Comment:
never do blank excepts. The code can then not even be `Ctrl-Ced`. It should work with the standard datatypes that we expect. It's fine if it raises another exception because new data was put in.